monitor/cli.py

```diff
@@ -123,7 +130,7 @@ def monitor(
             slack_token=slack_token,
             slack_channel_name=slack_channel_name
         )
-        success = data_monitoring.run(days_back, full_refresh_dbt_package)
+        success = data_monitoring.run(days_back, full_refresh_dbt_package, vars=config.dbt_vars_dict)
```

Comment:
We can pass a dict from `dbt_var` to `data_monitoring.run`, because the type of the parameter `vars` of `DbtRunner._run_command` is `Optional[dict]`. So, personally, it would be natural to parse in the `monitor` function in `cli.py`.

- https://github.com/elementary-data/elementary/blob/master/monitor/dbt_runner.py#L22